lib/plistLib/main.js

```diff
@@ -250,8 +252,8 @@ define([
             dict: function(doc, dict) {
                 var fragment = doc.createDocumentFragment(),
                     key, value, keyElement;
-                for(key in dict) { 
-                    if(!dict.hasOwnProperty(key) || dict[key] === undefined || dict[key] === null)
+                for(key in dict) {
+                    if(!Object.prototype.hasOwnProperty.call(dict, key) || dict[key] === undefined || dict[key] === null)
```

Comment:
Interesting. Trying to understand what that's a good thing, if you start with a {} primitive, you keep the hasOwnProperty, without any performance penalties (prototypes are singletons), what made you decide you needed a prototype-detached object?